src/main/java/com/stuypulse/robot/subsystems/Chimney.java

```diff
@@ -9,13 +9,21 @@
 
 public class Chimney extends SubsystemBase {
 
+    public enum ChimneyMode {
```

Comment:
I dont think you need chimney modes, in junhee's chart, the is ball detected is referring to the intake, not the chimney